gtnet/.ipynb_checkpoints/run-checkpoint.py

```diff
@@ -0,0 +1,48 @@
+from model import load_model
+from sequence import *
+import onnxruntime as rt
+import numpy as np
+import scipy.special
+import argparse
+
+#should this be in another file?
+def get_prob(x):
+    soft_out = scipy.special.softmax(x)
+    probabilities = soft_out.mean(axis=0)
+    return probabilities
+
+default_fasta_path = '/global/cscratch1/sd/azaidi/GCA_000006155.2_ASM615v2_genomic.fna'
```

Comment:
this looks leftover from testing?

Reply:
Removed + added error logging to handle missing fasta -- happy to modify as needed